assets/js/modules/search-console/components/dashboard/LegacyDashboardWidgetPopularKeywordsTable.js

```diff
@@ -37,18 +37,29 @@ import {
 } from '../../util';
 import { getCurrentDateRangeDayCount } from '../../../../util/date-range';
 import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site';
```

Comment:
This was the culprit of the failing tests and it not loading properly. We recently removed these exports from the main module and moved them to use `/constants`. Looks like this one needs updating, and then it's good to go 😄 
```suggestion
import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
```

Reply:
@tofumatt it seems to have worked 😃 . But the test still fails because of timeouts. Should I be worried about that? My other PR fails for the same reason. 🤔

Reply:
@kostyalmm It looks like it was just one of the 3 E2E jobs so I restarted it. These can be a bit unstable, but usually they all fail if there's something actually wrong. In rare cases, there is a WP version-related failure (since this is different between jobs) but you can usually tell by the error there too because the unstable tests always fail in the same way.